src/pages/logosValidator/dashboardDefinition.ts

```diff
@@ -246,7 +254,7 @@ const UNTYPED_LOGOS = {
   "fr:legumes-de-france": {
     tag: "fr:legumes-de-france",
     label: "Légumes de France",
-    logo: "https://static.openfoodfacts.org/images/lang/fr/labels/legumes-de-france.90x90.svg",
+    logo: "https://robotoff.openfoodfacts.org/api/v1/images/crop?image_url=https://images.openfoodfacts.org/images/products/325/622/009/4726/5.jpg&y_min=0.6817449927330017&x_min=0.25617462396621704&y_max=0.7520487904548645&x_max=0.4584151804447174",
```

Comment:
I don't think the query gets cached by robotoff, so we're putting additional load on it

Reply:
But `https://static.openfoodfacts.org/images/lang/fr/labels/legumes-de-france.90x90.svg` does not exist 🙈

Reply:
I merged too hastily. We might be inspired to start a cache of images we can iterate on faster than merging pixel perfect things to the server. WDYT ?